R/g.plot5.R

```diff
@@ -1,5 +1,6 @@
 g.plot5 = function(metadatadir=c(),dofirstpage=FALSE, viewingwindow = 1,f0=c(),f1=c(),overwrite=FALSE,
-                   metric="ENMO",desiredtz = "Europe/London") {
```

Comment:
Hi Matt, Did you turn off the flexibility to choose alternative metrics than ENMO because it messed up the visualisation? 
I think most people will be happy with ENMO as default, but I am wondering whether this could annoy those who prefer not to work with ENMO. For them ENMO will not be calculated in part1 causing g.plot5 to struggle finding it in the milestone data. 
So, the options I see are:
- ignore those users
- reverting this change
- keeping your edits but in addition modify lines 102-104 to select whatever metric is available in the milestone data, but not equal to "timestamps", "anglex", "angley" and "anglez".

Reply:
oh, right, I didn't consider other metrics used here. I actually implemented a check in the shell script to see that 'do.enmo=True' where the report will only get created if this is the case.